package.json

```diff
@@ -25,53 +25,55 @@
   },
   "homepage": "https://github.com/rapid7/propsd#readme",
   "devDependencies": {
-    "aws-sdk-mock": "~1.7.0",
-    "babel-cli": "^6.24.1",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.0",
-    "babel-preset-stage-2": "^6.24.1",
-    "chai": "~3.5.0",
-    "chai-as-promised": "~5.2.0",
-    "chokidar": "~3.0.0",
-    "coveralls": "~2.11.8",
-    "eslint": "~3.17.1",
-    "eslint-config-rapid7": "~2.9.0",
-    "eslint-plugin-import": "~2.2.0",
-    "eslint-plugin-react": "~6.10.0",
-    "istanbul": "~0.4.2",
-    "mocha": "~3.3.0",
-    "mocha-lcov-reporter": "~1.2.0",
-    "nock": "^9.0.13",
-    "nodemon": "^1.11.0",
-    "proxyquire": "~1.7.4",
-    "rimraf": "~2.5.2",
-    "s3rver": "2.2.9",
-    "should": "~8.1.1",
-    "should-sinon": "0.0.5",
-    "sinon": "~1.17.3",
-    "supertest": "~1.2.0",
-    "walk": "~2.3.9"
+    "@babel/cli": "7.8.4",
+    "@babel/core": "7.9.0",
+    "@babel/node": "7.8.7",
+    "@babel/plugin-proposal-class-properties": "7.8.3",
+    "@babel/plugin-proposal-decorators": "7.8.3",
+    "@babel/plugin-proposal-export-namespace-from": "7.8.3",
+    "@babel/plugin-proposal-function-sent": "7.8.3",
+    "@babel/plugin-proposal-numeric-separator": "7.8.3",
+    "@babel/plugin-proposal-throw-expressions": "7.8.3",
+    "@babel/plugin-transform-runtime": "7.9.0",
+    "@babel/preset-env": "7.9.0",
+    "@babel/preset-stage-2": "7.8.3",
+    "aws-sdk-mock": "5.1.0",
+    "babel-eslint": "10.1.0",
+    "chokidar": "3.3.1",
+    "coveralls": "3.0.11",
+    "eslint": "6.5.1",
```

Comment:
eslint-config-rapid7@2.11.1" has incorrect peer dependency "eslint@^3.0.0 || ^4.0.0

Reply:
Yeah, it's a known issue but doesn't affect how linting works. This is the only combination of updated `eslint` and `eslint-config-rapid7` that I was able to get working without errors. This is due to an issue in [rapid7/eslint-plugin/rapid7:sort-object-keys](https://github.com/rapid7/eslint-plugin-rapid7/blob/master/lib/rules/sort-object-keys.js#L5-L9) where it attempts to import an eslint lib that's no longer part of the project.